js/views/modals/Settings/Advanced.js

```diff
@@ -48,6 +49,29 @@ export default class extends baseVw {
     return super.getFormData(subset);
   }
 
+  clickResync() {
+    this.resynchronize();
+  }
+
+  resynchronize() {
+    this.getCachedEl('.js-resync').addClass('processing');
+    this.getCachedEl('.js-resyncComplete').addClass('hide');
+
+    this.resync = $.post(app.getServerUrl('wallet/resyncblockchain'))
+      .always(() => {
+        this.getCachedEl('.js-resync').removeClass('processing');
+      })
+      .fail((xhr) => {
+        const failReason = xhr.responseJSON && xhr.responseJSON.reason || '';
```

Comment:
Good catch. I didn't have it in because originally it looked like the call could take several minutes, and the user should know if it failed after they left the page. 

Now that it's working right, the call takes seconds or less, but I forgot to go back and add the abort check.